pagopa/io-app
# feat: [IA-673] Added versions history permission field

## Short description
This PR is going to add a permission field displayed to the user for the `versionsHistory` field on Zendesk. This permission field will replace the old `Version of the app in use`. The current app version will be sent anyway for backward compatibility.

<img width="421" alt="image" src="https://user-images.githubusercontent.com/5963683/154939023-4c6130f6-d9ca-44d7-9a31-3c431e75e2b9.png">

## List of changes proposed in this pull request
- Removed the `Version of the app in use` field
- Added the `versionsHistory` field
- For backward compatibility the current app version of the app is going to be sent anyway

## How to test
1. Open the contextual help and start writing a ticket
2. The `App version history` permission field should be displayed instead of the old `Version of the app in use`
3. Opening the ticket on Zendesk both the app version and the history should be available in the dashboard. This behaviour should be unchanged from the current app version.
